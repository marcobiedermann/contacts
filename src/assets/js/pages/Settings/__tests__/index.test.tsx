import { render } from '@testing-library/react';
import React from 'react';
import { MemoryRouter, Route } from 'react-router-dom';
import SettingsPage from '..';

describe('setting page component', () => {
  it('should render correctly', () => {
    expect.assertions(1);

    const { container } = render(
      <MemoryRouter>
        <Route component={SettingsPage} />
      </MemoryRouter>,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
