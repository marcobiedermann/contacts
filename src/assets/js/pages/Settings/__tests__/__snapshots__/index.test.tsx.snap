// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`setting page component should render correctly 1`] = `
<DocumentFragment>
  <div>
    <ul>
      <li>
        <button
          type="button"
        >
          en
        </button>
      </li>
      <li>
        <button
          type="button"
        >
          de
        </button>
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
